src/cli/types.ts

```diff
@@ -44,6 +44,21 @@ export type Config = {
 		 * @defaultValue `true`
 		 */
 		includeCreateClientInterface?: boolean;
+
+		/**
+		 * Determines if a `@prismicio/client` namespace named `Content` containing
+		 * all Document and Slice types should be included in the output.
+		 *
+		 * If set to `true`, a `Content` namespace from `@prismicio/client` will be
+		 * available to import to easily access types for your Prismic repository content.
+		 *
+		 * **Note**: If your project queries content from multiple Prismic
+		 * repositories, set `includeContentNamespace` to `false` and manually
+		 * import the content types instead.
```

Comment:
Can it still be left as `true` for the "main" repo?

Reply:
Yup, good call. I updated the description here: https://github.com/prismicio/prismic-ts-codegen/pull/28/commits/be23a7df948a92e3dcd615678723d84382597464